cypress/e2e/shared/about.test.ts

```diff
@@ -41,14 +41,12 @@ describe('About Page', () => {
 
     const newOrgName = `hard@knock.life${Math.random()}`
 
-    cy.intercept('PATCH', 'api/v2/orgs/*').as('patchOrg')
-
     cy.getByTestID('create-org-name-input').type(newOrgName)
     cy.getByTestID('rename-org-submit--button')
       .should('not.be.disabled')
       .click()
 
-    cy.wait('@patchOrg')
+    cy.intercept('PATCH', 'api/v2/quartz/orgs/*')
```

Comment:
does this exist within the quartz mock?

Reply:
That might account for the failure I was seeing. I always forget about quartz mock.